### PR TITLE
feat) add `clusterVersion, enableKubernetesAlpha, masterIpv4Cidr, pre…

### PR DESCRIPTION
**Pull request by chrisdostert**
…emptible` inputs

### CODE REVIEW

File: cmd.sh [hunk 1]
```diff
@@ -38,6 +38,9 @@ clusterCreateCmd=$(printf "%s --min-nodes %s" "$clusterCreateCmd" "$minNodes")
 if [ "$async" = "true" ]; then
   clusterCreateCmd=$(printf "%s --async" "$clusterCreateCmd")
 fi
+if [ "$clusterVersion" != " " ]; then
```
Comment (luisgarciaalanis) [inline cmd.sh]: why not keep vars outside quotes?
if [ $clusterVersion != " " ]; then

**Review by luisgarciaalanis — APPROVED** [review]

File: cmd.sh (hunk 1 above)
Comment (chrisdostert) [reply cmd.sh]: might have special characters

File: op.yml [hunk 2]
```diff
@@ -5,6 +5,11 @@ inputs:
     boolean:
       description: don't wait for the op to complete
       default: false
+  clusterVersion:
+    string:
+      default: ' '
```
[the remark this answers is not in the document]
Comment (chrisdostert) [reply op.yml]: space

[the remark this answers is not in the document]
Comment (chrisdostert) [reply op.yml]: it's entirely because we don't support optional parameters; empty string would fail validation

**Pull request merged**